library/src/plugins/attributes/bind.ts

```diff
@@ -119,6 +122,7 @@ export const Bind: AttributePlugin = {
                   },
                 ),
               )
+              target.dispatchEvent(new CustomEvent(evt.type, {detail: true}))
```

Comment:
Hmm, maybe this should go in a `finally` so the event doesn't get swallowed in case there is an error reading the files.